tox.ini

```diff
@@ -1,42 +1,66 @@
-# Tox (https://tox.readthedocs.io/) is a tool for running tests in
-# multiple virtualenvs. To use it, "pip install tox" and then run
-# "tox" from this directory.
-
 [tox]
-skipsdist = true
 # mypy doesn't current pass for us on Windows. Fixing that is being tracked by
 # https://github.com/certbot/certbot/issues/7803.
-envlist = {cover,lint}-{win,posix},mypy-posix
+envlist = {cover,lint}-{win,posix},mypy
```

Comment:
https://eff-certbot.readthedocs.io/en/stable/contributing.html#submitting-a-pull-request mentions:

>Run tox --skip-missing-interpreters to run the entire test suite including coverage

This is what runs for me (possibly we have the same problem on `master` so feel free to ignore this for now):

    _________________________________________________________ summary _________________________________________________________
    SKIPPED:  cover-win: platform mismatch ('darwin' does not match 'win32')
      cover-posix: commands succeeded
    SKIPPED:  lint-win: platform mismatch ('darwin' does not match 'win32')
      lint-posix: commands succeeded
      mypy: commands succeeded
      congratulations :)

Reply:
This is "a feature, not a bug" 🙃 See https://tox.wiki/en/legacy/example/platform.html#platform-specification.

We do have the same behavior on `master` though so I'd rather not do anything about it here, but I'm open to ideas to make tox's behavior and/or our docs more comprehensible.

Reply:
Sorry, I meant that the unit tests (i.e. `tox -e py`) do not seem to run by default. That seems in contradiction to the part of the page I quoted.

Reply:
Oh! Yeah we currently have this behavior on master too. I tweaked the wording slightly.

If we want to do more here, I think we may want to wait until https://github.com/certbot/certbot/pull/9669 is merged. That removes most of the differences between `cover` and `py` at which point maybe we should just remove `cover` in favor of the default environment `py` which is what we currently do in josepy: https://github.com/certbot/josepy/blob/09d9ffed0d04f9558b546259f5da7cb99ae07297/tox.ini#L24